src/borg/helpers/parseformat.py

```diff
@@ -882,7 +882,7 @@ def __init__(self, archive, format, *, json_lines=False):
         self.used_call_keys = set(self.call_keys) & self.format_keys
 
     def format_item_json(self, item):
-        return json.dumps(self.get_item_data(item), cls=BorgJsonEncoder) + "\n"
+        return json.dumps(self.get_item_data(item), cls=BorgJsonEncoder, sort_keys=True) + "\n"
```

Comment:
same.

Reply:
@ThomasWaldmann the enhancement request in the original issue ticket asked that the order of the response be consistent in terms of order (preferably alphabetically ordered). That's why I added those two in.

Reply:
OK, but please mention this change in the commit comment and also please add ", fixes #XXXX" to refer back to the ticket you are fixing.

Reply:
the commit comment change is still missing.

maybe you can do a `git rebase -i master` at some time and add that (and also collapse some fixup commits).